applications/StructuralMechanicsApplication/custom_python/structural_mechanics_python_application.cpp

```diff
@@ -346,6 +349,15 @@ PYBIND11_MODULE(KratosStructuralMechanicsApplication,m)
     // for DEM-FEM 2D
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, IMPOSED_Z_STRAIN_OPTION)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, IMPOSED_Z_STRAIN_VALUE)
+
+    // The ratios between the yield strength in the isotropic space and the anisotropic space
+    // at each direction in local coordinates ratio_x = ft / ft,x 
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, ISOTROPIC_ANISOTROPIC_YIELD_RATIO_X );
```

Comment:
I think it is better to directly register a symmetric tensor (there is a macro for that)